java/src/main/java/com/saucelabs/simplesauce/RemoteDriverInterface.java

```diff
@@ -6,6 +6,6 @@
 import java.net.MalformedURLException;
 
 public interface RemoteDriverInterface {
```

Comment:
We should rename this to something else--part of the point of Interfaces is that users ideally wouldn't know or care that they're using an interface... this is the Object of Record that they care about. I also don't like "RemoteDriver", so we're going to have to come up with something else. It's not an emergency.

Reply:
@mmerrell Thanks for pointing that out. I'm happy to make changes to make my code as clean as possible. I try to avoid code rot and avoid Leblanc's Law. 
What this interface does is it instantiates a new RemoteWebDriver if it's the concrete implementation. And returns a fake for testing.
The reason why I added "_Interface_" to it is that it's really confusing as to which object I was interacting with. i was unable to look at the code and understand whether I'm using an interface or a concrete class. In C#, we put an "_I_" in front of our interface so that it's easy to look at the code and understand what the class is inheriting from. I don't know if that's a bad practice in Java? Also, maybe this confusion was a result of poor naming...
Side note. Our end uses would not know about this interface regardless. But the developers of the code would.

Some name ideas: RemoteDriverManager, RemoteDriverCreator, or something else?